RAD-Seq/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Galaxy - RAD-Seq training material
+
+FROM engineson/galaxy-rad-seq
```

Comment:
yes, but I have to correct / polish / finish it!!!! There is a lot of mistake on it...